opensearch/src/main/java/org/opensearch/sql/opensearch/storage/scan/OpenSearchIndexScan.java

```diff
@@ -47,14 +47,18 @@ public class OpenSearchIndexScan extends TableScanOperator implements Serializab
   /** Search response for current batch. */
   private Iterator<ExprValue> iterator;
 
-  private Settings pluginSettings;
+  private boolean fieldTypeTolerance;
 
   /** Creates index scan based on a provided OpenSearchRequestBuilder. */
   public OpenSearchIndexScan(
-      OpenSearchClient client, int maxResponseSize, OpenSearchRequest request) {
+      OpenSearchClient client,
+      boolean fieldTypeTolerance,
+      int maxResponseSize,
+      OpenSearchRequest request) {
```

Comment:
nit: we could keep original constructor which means fieldTypeTolerance=true, it could avoid unnecessary UT chagne.

Reply:
+1. I'm also curious if it even needs to be a parameter since every refactor hardcodes it to `true` and the `false` case isn't even tested. What's the use case where we don't want this?